### DISCUSSION TITLE
Accessibility issue

### DISCUSSION THREAD

**Discussion by hynits** [rustdesk/rustdesk#6584] [General]
### Bug Description

This is an interactive flat panel with Android 13, and I have installed RustDesk version rustdesk-1.2.3-aarch64-signed.apk.

https://github.com/rustdesk/rustdesk/assets/152599583/a657535a-c4d7-4e8e-902c-08613ccb66fb

However, I can't enable the input control.

### How to Reproduce

Install Rustdesk version rustdesk-1.2.3-aarch64-signed.apk.
And try to enable "input control" option 

### Expected Behavior

Need to enable the option "input control"

### Operating system(s) on local side and remote side

Android 13

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

![MicrosoftTeams-image (10) 1](https://github.com/rustdesk/rustdesk/assets/152599583/56118ba2-a09d-4a47-9474-c1edc2b70165)
![MicrosoftTeams-image (9) 1](https://github.com/rustdesk/rustdesk/assets/152599583/cac79458-5193-4828-bca2-30d5db669c3b)

**Comment by deep-soft** [comment]
https://github.com/rustdesk/rustdesk/discussions/6241
